app/indexers/hyrax/file_set_indexer.rb

```diff
@@ -62,5 +64,18 @@ def file_format
           object.format_label
         end
       end
+
+      def human_readable_label_name(parent)
+        return unless parent
+
+        parent_title = parent.title.first
+        page_number = parent_title[/Page \d+/]
```

Comment:
Could you leave a comment as to why this?  Where's this magic regex coming from?  How will future us debug this when we change our title function?